met-api/src/met_api/services/email_verification_service.py

```diff
@@ -42,10 +42,18 @@ def get_active(cls, verification_token):
     def create(cls, email_verification: EmailVerificationSchema, session=None) -> EmailVerificationSchema:
         """Create an email verification."""
         cls.validate_fields(email_verification)
-        email_address = email_verification.get('email_address')
+        email_address: str = email_verification.get('email_address')
+        survey = SurveyModel.get_open(email_verification.get('survey_id'))
+        engagement: EngagementModel = EngagementModel.find_by_id(survey.engagement_id)
+        if engagement.is_internal and not email_address.endswith('@gov.bc.ca'):
```

Comment:
@gov.bc.ca ; can you move this to a constant?

Reply:
done